diff --git a/src/ComponentWithFormTrait.php b/src/ComponentWithFormTrait.php
index 806cfaa..dc0ce17 100644
--- a/src/ComponentWithFormTrait.php
+++ b/src/ComponentWithFormTrait.php
@@ -146,7 +146,9 @@ trait ComponentWithFormTrait
     private function submitForm(bool $validateAll = true): void
     {
         if (null !== $this->formView) {
-            throw new \LogicException('The submitForm() method is being called, but the FormView has already been built. Are you calling $this->getForm() - which creates the FormView - before submitting the form?');
+            $this->shouldAutoSubmitForm = false;
+            $this->form = null;
+            $this->formView = null;
         }
 
         $form = $this->getForm();
