diff --git a/assets/dist/turbo_stream_controller.js b/assets/dist/turbo_stream_controller.js
index b1eeaed..287dbbf 100644
--- a/assets/dist/turbo_stream_controller.js
+++ b/assets/dist/turbo_stream_controller.js
@@ -16,7 +16,7 @@ class default_1 extends Controller {
     }
     connect() {
         if (this.url) {
-            this.es = new EventSource(this.url);
+            this.es = new EventSource(this.url, {withCredentials: true});
             connectStreamSource(this.es);
         }
     }
